blueman/main/Sendto.py

```diff
@@ -94,11 +95,17 @@ def __init__(self, device, adapter_path, files):
         self.num_files = len(self.files)
         try:
             self.client = obex.Client()
-        except obex.ObexdNotFoundError:
```

Comment:
This can get raised in `Client.__init__` in case an old obexd (for BlueZ 4) is found. It looks like I ran into that situation when working on #259. I'm fine with ignoring the case, but then we should remove the related code from `Client.__init__`.

Reply:
Yeah, its not relevant in bluez5 anymore. I'll drop that in a separate commit later tonight (first have something to eat). Not on my best game lately :disappointed: .

Reply:
Should be good now. Will be testing a bit more this week. Off to get some :coffee:.